scripts/setup-ci-environment.sh

```diff
@@ -54,17 +55,19 @@ if [ ! -d .cache/intellij/$FULL_IJ_BUILD_NUMBER/plugins ]; then
   mkdir -p plugins
   pushd plugins
 
-  # wget -q --no-check-certificate -O Scala.zip "https://plugins.jetbrains.com/pluginManager/?action=download&id=$SCALA_PLUGIN_ID&build=$FULL_IJ_BUILD_NUMBER"
   if [ "$SCALA_PLUGIN_CHANNEL" == "stable" ]; then
-      wget -q -O Scala.zip "https://plugins.jetbrains.com/plugin/download?pluginId=$SCALA_PLUGIN_ID&version=$SCALA_PLUGIN_VERSION"
+      curl -LSso Scala.zip "https://plugins.jetbrains.com/plugin/download?pluginId=$SCALA_PLUGIN_ID&version=$SCALA_PLUGIN_VERSION"
   else
-      wget -q -O Scala.zip "https://plugins.jetbrains.com/plugin/download?pluginId=$SCALA_PLUGIN_ID&version=$SCALA_PLUGIN_VERSION&channel=$SCALA_PLUGIN_CHANNEL"
+      curl -LSso Scala.zip "https://plugins.jetbrains.com/plugin/download?pluginId=$SCALA_PLUGIN_ID&version=$SCALA_PLUGIN_VERSION&channel=$SCALA_PLUGIN_CHANNEL"
   fi
-  (echo "$SCALA_PLUGIN_SHA Scala.zip" | sha256sum -c -) && unzip -q Scala.zip
+  curl -LSso python.zip "https://plugins.jetbrains.com/pluginManager/?action=download&id=$PYTHON_PLUGIN_ID&build=$FULL_IJ_BUILD_NUMBER"
+
+  sha256sum --strict -c ../scripts/checksums.txt
```

Comment:
Yes, that's right. However:
1. This command has been used in the previous version of the script as well;
2. This is probably not the only incompatibility with Mac present in these scripts;
3. The GNU coreutils can be installed on Mac using Homebrew;
4. I am planning to do a major refactoring of the build in the future.

I would like to have this pull request merged as it is, because it already solves a couple problems, and address the rest of the problems later.

Reply:
Sounds good. Thanks for clarifying!